e2e/linodecluster-controller/minimal-linodecluster/chainsaw-test.yaml

```diff
@@ -4,6 +4,9 @@ kind: Test
 metadata:
   creationTimestamp: null
   name: minimal-linodecluster
+  # Label to trigger the test on every PR
```

Comment:
while we are labeling these workflows can we also add a label for what component they are part of like `linodecluster:` or `linodevpc:` so we could theoretically have a test filter that just triggers one of these

Reply:
I can add them to all the tests